mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAModel.scala

```diff
@@ -287,15 +408,26 @@ class DistributedLDAModel private (
     private[clustering] val globalTopicTotals: LDA.TopicCounts,
     val k: Int,
     val vocabSize: Int,
-    private[clustering] val docConcentration: Double,
-    private[clustering] val topicConcentration: Double,
+    protected[clustering] val docConcentration: Vector,
```

Comment:
I don't think it's necessary since the constructor is private and is only created from the auxillary constructor, which replicates `docConcentration` `k` times